app/assets/javascripts/discourse/routes/discovery.js.es6

```diff
@@ -13,8 +13,21 @@ export default DiscourseRoute.extend(OpenComposer, {
   },
 
   beforeModel(transition) {
-    const user = User;
+    // the new bookmark list is radically different to this topic-based one,
+    // including being able to show links to multiple posts to the same topic
+    // and being based on a different model. better to just redirect
     const url = transition.intent.url;
+    if (
+      this.siteSettings.enable_bookmarks_with_reminders &&
+      url === "/bookmarks"
+    ) {
+      this.transitionTo(
+        "userActivity.bookmarksWithReminders",
+        this.currentUser
+      );
+    }
+
+    const user = User;
```

Comment:
This is slightly confusing to me - `user` in lower case usually indicates an local or member variable. Could the code below use `User` instead of assigning it like this?

Reply:
This was there already, not sure why it is assigned, will clean up